usaspending_api/download/filestreaming/download_generation.py

```diff
@@ -120,6 +122,7 @@ def generate_download(download_job):
         # Remove generated file
         if not settings.IS_LOCAL and os.path.exists(zip_file_path):
             os.remove(zip_file_path)
+        _kill_spawned_processes(download_job)
```

Comment:
Could there be spawned processes still running here?

Reply:
It'd be rare but I think possible. 

What I realized is that the queue framework should stop trying to be responsible for taking care of spawned child processes in the "processing-app code" that it is dispatching to. That should be a cleanup responsibility of the processing-app. In this case, since the download code spawns sub-processes, it should handle cleanup of them in case of an error. 

Now, it's likely that the error is going to either be in psql, in which case it will kill itself, or in zip, which won't run til after psql, and will similarly kill itself. So this would only occur if the python code around psql or the zip (e.g. in the wait_for_process) somehow encountered an error. So that's why this was added to the finally as a just-in-case step.